<launch>
  <node name="filtered_world" pkg="roboteam_world" type="filtered_world" launch-prefix="xterm -e gdb --args"/>
</launch>
